Show pipeline last updated tooltip on pipeline name hover

analytically opened the pull request:
Add tooltip to pipeline breadcrumb name showing "pipeline last updated at <datetime>" when user hovers over the pipeline name in the top bar.

taylorsilva reviewed (commented):
Works fine on instance groups too:
<img width="678" height="110" alt="image" src="https://github.com/user-attachments/assets/19511668-1040-4883-85fe-4b6645fd68b3" />

Looks like we're just missing tests. I think they would go here: https://github.com/concourse/concourse/blob/master/web/elm/tests/PipelineTests.elm
and
https://github.com/concourse/concourse/blob/master/web/elm/tests/TopBarTests.elm

Looks good otherwise!

analytically committed:
Show pipeline last updated tooltip on pipeline name hover

Add tooltip to pipeline breadcrumb name showing "pipeline last updated on <date> at <time>"
when user hovers over the pipeline name in the top bar.

Signed-off-by: Mathias Bogaert <mathias.bogaert@gmail.com>
